src/plugin_discovery/errors.js

```diff
@@ -0,0 +1,63 @@
+
+const $code = Symbol('FindErrorCode');
```

Comment:
Is this a convention we want to adopt? When `$` precedes, it means Symbol, otherwise it means Stream/Observable?

Reply:
Good catch, I don't think we want to have both symbols and streams tagged with $, `errorCodeProperty` is probably better